GoogleChromeLabs/ps-analysis-tool
# Feature: Add visited index and add dropdown icon to open non-animated flow.

## Description
This PR aims to add visited index in the interactive mode and adds a dropdown button to show the flow without animation.

## Relevant Technical Choices



## Testing Instructions
- Clone this branch
- In the terminal run `npm i && npm run dev:ee:pa`.
- Now in the browser, switch to interactive mode.
- Once flow is completed, it should show the index of visit.
- Now in the visited index there will be a double arrow drop down click on that a non-animated flow will open.

## Additional Information:



## Screenshot/Screencast



---

## Checklist



~~- [ ] This code is covered by unit tests to verify that it works as intended.~~
~~- [ ] The QA of this PR is done by a member of the QA team (to be checked by QA).~~



Partially addresses #857
